hips/utils/wcs.py

```diff
@@ -124,6 +124,30 @@ def create(cls, skydir: SkyCoord, shape: tuple, coordsys: str = 'CEL',
 
         return cls(w, shape)
 
+    @classmethod
+    def create_simple(cls, skydir: SkyCoord, shape: tuple, fov_deg: float,
+                      coordsys: str = 'CEL', projection: str = 'AIT') -> 'WCSGeometry':
+        """Create WCS object programmatically using field of view (`WCSGeometry`).
+
+        Parameters
+        ----------
+        skydir : `~astropy.coordinates.SkyCoord`
+            Sky coordinate of the WCS reference point
+        shape : `tuple`
+            Shape of the image (Numpy axis order: y, x)
+        fov_deg : `float`
```

Comment:
I would suggest to change this option to `fov` and then in the code do this:
```
from astropy.coordinates import Angle
fov = Angle(fov)
```
This will force the user to be explicit about units of inputs (lead to more readable user scripts) and allow e.g. to pass strings like "3 deg" (i.e. users don't have to instantiate an Angle object before calling.